src/CommandRunner.cpp

```diff
@@ -355,5 +322,18 @@ Try<string> CommandRunner::run(const Command& command,
     return Error(e.what());
   }
 }
+
+Try<string> CommandRunner::run(const Command& command,
+                               const std::string& input) {
+  Future<Try<string>> output = asyncRun(command, input);
+  auto result = await(output);
+  if (!result.await(Seconds(command.timeout() + 1))) {
```

Comment:
Why ?

Reply:
I wanted to ensure that in the typical timeout case, the `after()` in `asyncRun()` is triggered before this `await()`expires. But I changed my copy, and now rely only to `asyncRun()` for timeout.